gui/message.cpp

```diff
@@ -26,6 +26,7 @@
 #include "gui/gui-manager.h"
 #include "gui/ThemeEval.h"
 #include "gui/widget.h"
+#include <common/translation.h>
```

Comment:
Please use `""` style includes for in-project files instead of `<>`.

Reply:
Agreed.
I added this line with some VS auto magic - weird that it used the `<>` style.